Makefile

```diff
@@ -1,7 +1,7 @@
 include Makefile.common
 
 TOP_DIR := $(dir $(abspath $(firstword $(MAKEFILE_LIST))))
-override CFLAGS += -I include
+override CFLAGS += -I include -DNEED_GET_DATA
```

Comment:
No, not add one config option, since extra check on io->flags should have zero cost.